run-backend.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/ash
+
+export GDAL_LIBRARY_PATH='/usr/lib/libgdal.so.34'
+export GEOS_LIBRARY_PATH='/usr/lib/libgeos_c.so.1'
```

Comment:
The setting of `GDAL_LIBRARY_PATH` and `GEOS_LIBRARY_PATH` should be added to `uipa_org/settings/development.py` to set these values or be overridden by shell environment variables like the ELASTICSEARCH_* variables.

This would help with exec'ing into the app container to manual create an admin user which fails when run from this script and to manually load the seed data so that there are FOI laws, jurisdications, public agencies, and tags for the public agencies.

Reply:
Those env variables are needed for the module imports to work.  I'm not sure how the settings can be added to `uipa_org/settings/development.py` .  I'd be happy to add it if you let me know how to set that up.

Reply:
In the `Dev` class, you can add the following above `ELASTICSEARCH_HOST`:
```
GDAL_LIBRARY_PATH = values.Value("/usr/lib/libgdal.so.34", environ=True, environ_name="GDAL_LIBRARY_PATH")
GEOS_LIBRARY_PATH = values.Value("/usr/lib/libgeos_c.so.1", environ=True, environ_name="GEOS_LIBRARY_PATH")
```

Reply:
Changed, thought the gdal and geos libs were being imported much earlier in the setup process